### ISSUE TITLE
Gantt chart lines with "crispEdges" don't scale well

### ISSUE THREAD

**Issue by stevenengler** [knsv/mermaid#988]
**Describe the bug**
Gantt chart "tick" lines are generated with the style property `shape-rendering: crispEdges`. This causes them to render with different sizes in Firefox and Chromium.

**To Reproduce**
Use the example Gantt chart from the README and scale the svg. The screenshot below was generated by enclosing the `<div class="mermaid"></div>` element in a `<figure>` tag, and scaling the figure.

**Expected behavior**
The lines should have equal width. Removing the `shape-rendering` property fixes this, but I don't know if it was added for a specific reason.

**Screenshots**
![gantt-chart](https://user-images.githubusercontent.com/3708797/66520898-0a590900-eab8-11e9-9748-eefa1d66ed8b.png)

(note how the vertical lines have different sizes)

**Desktop:**
 - OS: Linux
 - Browser: Firefox, Chromium

**Comment by simpltek** [comment]
Are you viewing this on a High DPI screen? Maybe the crispness is not visually accurate on non-High DPI screens :/

**Comment by stevenengler** [comment]
It's 1080p, but I think that only makes the problem more noticeable; it would still be incorrect on higher resolutions. It would be fine if all of the lines were rendered in the same way, but since only the browser can control that, my recommendation would be to render them normally.

**Comment by jgreywolf** [comment]
Can someone confirm for me if this is still an issue? As there have been many changes in the past 4 years ;)

_I will close this after 30 days if no response_

**Comment by stevenengler** [comment]
Yes this still exists. You can see it in the [example](https://mermaid.live/edit#pako:eNp90cGOgyAQBuBXIZxtFbG29bbZ3fsmvXKZylhJEAyOTZrGd1_sto3xsHMBhu-HBO689hp5xS_giJQbsCbjHTv9jcp9-q63SKhZpb3DhMXSOIiE5ZkoNpnYZGXynh6U-4jBK7JnVfBYJo9QvgjtEya1cj8QwFq0TMz4lZqxTBg0hOF5m1jifI2Lf7Bc490CyxUu1rhc4GLGPOEdhg6Mjq92V44xxanFDhWv4lRjA6MlxZWbIh17DYTf2pAPvGrADphwGMmfbq7mFYURX-jLwCVA91bWg8YYunO69Y8vMgPFI2vvGnOZ-2Owsd0S9UOVpvP29mKoHc_b2nfpYHQLgdrrsUzLvDxALrHcS9hJqeuzOB6avBCN3mciBz5N0y_wxZ0J) from the README if you enable "pan & zoom" and then change the zoom level.
